google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -639,6 +639,9 @@ def rotate encryption_key: nil, new_encryption_key: nil
         # Permanently deletes the file.
         #
         # @return [Boolean] Returns `true` if the file was deleted.
+        # @param [Boolean,Integer] generation Select a specific revision of the
+        #   file to delete. When `true` it will use the object's generation. The
```

Comment:
Sorry, I don't understand. What is the difference between passing `generation: true` and not passing `generation` at all?